### PR TITLE
Depreacted: ListItem/Alphi Complete no Android file problem

### PR DESCRIPTION
**Pull request by aahenry51**

### CODE REVIEW

File: src/components/ListItem/ListItem.view.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}) => {
+  if (Title == '') {
+    Title = 'Test 1';
+  } else if (Title.length > MaxTitleLength) {
+    Title = '...';
+  }
+
+  return (
+    <View style={style.container}>
+      <TouchableOpacity onPress={onPress}>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:27]: <TouchableOpacity onPress={onPress} style={style.container}>

File: src/components/ListItem/ListItem.view.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <View style={style.AlignmentContainer}>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:28]: This view is extra

File: src/components/ListItem/ListItem.view.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <View style={style.AlignmentContainerText}>
+            <Text style={style.FontStyleSizeTitle}>{Title}</Text>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:30]: use camel case for {Title} prop

File: src/components/ListItem/ListItem.view.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    Title = 'Test 1';
+  } else if (Title.length > MaxTitleLength) {
+    Title = '...';
+  }
+
+  return (
+    <View style={style.container}>
+      <TouchableOpacity onPress={onPress}>
+        <View style={style.AlignmentContainer}>
+          <View style={style.AlignmentContainerText}>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:29]: Set different name for text container

File: src/components/ListItem/ListItem.view.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            <Text style={style.FontStyleSizeTitle}>{Title}</Text>
+            <Text style={style.FontStyleSubTitle}>{subtitle}</Text>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:31]: export as `styles`

`style`=>`styles`

File: src/components/ListItem/ListItem.view.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+          </View>
+
+          <View style={style.IconStyle}>
```
Comment (lasharela) [inline src/components/ListItem/ListItem.view.js:34]: style.IconStyle => styles.IconContainer

File: src/components/ListItem/ListItem.Style.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    paddingTop: 11,
+    paddingLeft: 18,
+    fontSize: 18,
+    color: '#1E4D58',
+  },
+
+  FontStyleSubTitle: {
+    fontFamily: 'Poppins',
+    fontWeight: 'normal',
+    paddingTop: 5,
```
Comment (lasharela) [inline src/components/ListItem/ListItem.Style.js:34]: find closes sizes in `size` object.

File: src/components/ListItem/ListItem.Style.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {StyleSheet} from 'react-native';
+
+export const style = StyleSheet.create({
+  container: {
+    backgroundColor: '#ffffff',
```
Comment (lasharela) [inline src/components/ListItem/ListItem.Style.js:6]: replace color.

**Review by lasharela — APPROVED** [review]

**Review by raynesmike — APPROVED** [review]
I Approve this

**Pull request closed without merging**